pkg/favirecon/db.json

```diff
@@ -1004,6 +1004,10 @@
     "1999746265": "ispCP Omega Virtual Hosting Control System",
     "2006716043": "Intelbras SA",
     "2019488876": "Dahua Storm (IP Camera)",
```

Comment:
The favicons are not correctly ordered, see above the ones you added there is `2019...`. Please reorder them